lib/commands/debug/java-debug.js

```diff
@@ -0,0 +1,25 @@
+const AbstractDebugFlow = require('./abstract-debug-flow');
+
+class JavaDebugFlow extends AbstractDebugFlow {
+    static canHandle(runtime) {
+        return runtime.includes('java');
+    }
+
+    constructor({ skillInvocationInfo, waitForAttach, debugPort, token, skillId, debugRegion }) {
+        let execCmd = `cd ${skillInvocationInfo.skillCodeFolderName}; mvn exec:exec -Dexec.executable=java -Dexec.args=`
+        + '"-classpath %classpath com.amazon.ask.localdebug.LocalDebuggerInvoker "'
+        + `"--accessToken ${token} --skillId ${skillId} --skillStreamHandlerClass ${skillInvocationInfo.handlerName} --region ${debugRegion}"`;
+        if (waitForAttach) {
+            execCmd = `cd ${skillInvocationInfo.skillCodeFolderName}; mvn exec:exec -Dexec.executable=java -Dexec.args=`
+            + `"-classpath %classpath -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=${debugPort} "`
+            + `"com.amazon.ask.localdebug.LocalDebuggerInvoker --accessToken ${token} --skillId ${skillId} --region ${debugRegion}"`
+            + `"--skillStreamHandlerClass ${skillInvocationInfo.handlerName}"`;
```

Comment:
should there be only one pair of quotes surronding this entire string? now there are four pairs